### ISSUE TITLE
The bug clerk is mistaken about unassignement

### ISSUE THREAD

**Issue by dmlloyd** [jboss-set/bug-clerk#105]
In [JBEAP-14241](https://issues.jboss.org/browse/JBEAP-14241?page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel&focusedCommentId=13542831#comment-13542831) the bug clerk has said "This issue is still unassigned, for over two weeks,  while having a high priority".

It is true that it is unassigned.  It is also correct that the priority of the bug is Major, and that the bug is over two weeks old.  However, I only unassigned it minutes ago.

**Comment by rpelisse** [comment]
Yep, given how this check is done this is definitely a trap it can fall into. That being said, the underlying issue (nobody assigned to the issue) is still correct. The only mistake of Bugclerk is to say "still" instead of "again". 

Note that it could be fix by adding some analysis of the issue history to the check, but this would be quite cumbersome, for something that seems to be not occuring that often. However being doing so, I would wait for a bit more report or complains about this specific issue. 

Sidenote: as this is a JBEAP issue, why not simply assigned it to the JBoss SET rather to unassigned?
